.gitignore

```diff
@@ -51,4 +51,6 @@ ci/authn-k8s/dev/policies/*.yml
 # Vim backup files
 *.sw[po]
 
-
+# Ignore link files sourced at audit engine DB migration 
+db/migrate/*_create_messages_table.conjur_audit.rb
+db/migrate/*_add_indexes.conjur_audit.rb
```

Comment:
add newline

Reply:
There is already new line in the end of file

Reply:
done